src/coins.h

```diff
@@ -43,7 +43,12 @@ class Coin
 
     //! construct a Coin from a CTxOut and height/coinbase information.
     Coin(CTxOut&& outIn, int nHeightIn, bool fCoinBaseIn) : out(std::move(outIn)), fCoinBase(fCoinBaseIn), nHeight(nHeightIn) {}
-    Coin(const CTxOut& outIn, int nHeightIn, bool fCoinBaseIn) : out(outIn), fCoinBase(fCoinBaseIn),nHeight(nHeightIn) {}
+    Coin(const CTxOut& outIn, int nHeightIn, bool fCoinBaseIn) : out(outIn), fCoinBase(fCoinBaseIn), nHeight(nHeightIn) {}
+
+    Coin(Coin&& other) noexcept = default;
+    Coin& operator=(Coin&&) noexcept = default;
+    Coin(const Coin&) noexcept = default;
+    Coin& operator=(const Coin&) noexcept = default;
```

Comment:
Is the reason that all of these default constructors can be declared noexcept that all of `Coin`'s members' respective default constructors that get invoked are noexcept?